patches/chrome-browser-ui-views-bookmarks-bookmark_bubble_view.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+index ccd0049235285c1dc058936b09f2aaae9f25ace1..b6173b3534c58d633a72fc335763fdc887a1dc48 100644
+--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
++++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+@@ -70,7 +70,7 @@ views::Widget* BookmarkBubbleView::ShowBubble(
+   // Bookmark bubble should always anchor TOP_RIGHT, but the
+   // LocationBarBubbleDelegateView does not know that and may use different
+   // arrow anchoring.
+-  bookmark_bubble_->set_arrow(views::BubbleBorder::TOP_RIGHT);
++  bookmark_bubble_->set_arrow(views::BubbleBorder::TOP_LEFT);
```

Comment:
I think you can use chromium_src override here and rename `TOP_RIGHT` to `TOP_LEFT` w/ `#define`
You'll need some includes at the top as usual from `chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc` inside the `chromium_src` override.

Reply:
I'm more concerned about the comment above it

Reply:
// Bookmark bubble should always anchor TOP_RIGHT, but the
  // LocationBarBubbleDelegateView does not know that and may use different
  // arrow anchoring

Reply:
so it seems like we shouldn't change this and also that there is another way to set it to LEFT

Reply:
@bridiver I think that comment is saying that `LocationBarBubbleDelegateView` may have another default, or may calculate anchor point based on window width, etc, but for the Bookmark version of that delegate it should always be `TOP_RIGHT` for Chrome (or `TOP_LEFT` for Brave), which would mean this would be the correct place to specify this. We do not want to modify the anchor point for any other bubble views.

@bbondy I'll try the chromium_src method

Reply:
As discussed with @bridiver, we cannot redefine the enum value using the pre-processor. So instead I tried to re-call `set_arrow` in our own code via the following:
```
bubble_widget->widget_delegate()->AsBubbleDialogDelegate()->set_arrow(views::BubbleBorder::TOP_LEFT);
```
But this value is used during `Init`, which is called during `CreateBubble`, which has already been called by the time the bubble is returned to us.

So the only other option would be to override `BookmarkBubbleView::ShowBubble` but there is quite a bit of functional logic in that method, which seems more risky than this small patch.